buildpipeline/pipelines.json

```diff
@@ -425,7 +438,42 @@
             "Type": "build/product/",
             "PB_BuildType": "Release"
           }
-        }
+        },
```

Comment:
You don't need to add these 2 jobs - just add "redhat.69.amd64" to the list of target queues in the pre-existing Linux buildtest jobs. The reason for this is at the tests are still being built on Windows, so we don't need to do anything special for RHEL other than run the tests in the right queue.